cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/commands/ApplicationArgs.java

```diff
@@ -22,6 +21,8 @@
 
 public class ApplicationArgs {
 
+    private static final String DYNAMIC_PARAMETER_CHANGES_ONLY = "changes_only";
```

Comment:
maybe it is worth showing this as a regular parameter to get more visibility. WDYT?